src/oatpp/core/data/mapping/type/Any.cpp

```diff
@@ -38,7 +38,10 @@ Any::Any(std::nullptr_t) : Any() {}
 
 Any::Any(const std::shared_ptr<AnyHandle>& handle, const Type* const type)
   : ObjectWrapper(handle, __class::Any::getType())
-{}
+{
+  //TODO: is there indeed no use for |type| parameter passed above?
```

Comment:
As an `ObjectWrapper`, `Any` must have this constructor. It is used in `ObjectWrapper.staticCast<T>(...)` method.
However, unlike other `ObjectWrapper`s `Any` won't change its `valueType`.
`Any` is always object wrapper above `AnyHandler`.

Reply:
I'm moving your comment to the code then.